layouts/shortcodes/app_and_api_protection_java_setup_options.md

```diff
@@ -0,0 +1,9 @@
+<div class="alert alert-info">
+<p>You can enable App and API Protection for Java services with the following setup options:</p>
+
+<ol>
+<li>If your Java service already has APM tracing set up and running, then skip to <a href="#automatically-enabling-app-and-api-protection-through-remote-configuration">service configuration</a></li>
+<li>If your Java service doesn't have APM tracing set up, you can easily enable App and API Protection with Datadog's <a href="https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/single-step-apm/{{ .Get "platform" }}/">Automatic Installation</a></li>
```

Comment:
I commented the same in the Ruby's documentation PR, in this line we are talking about "enable Ap and API Protection" but the content of the link is talking only about APM enablement, not AAP.

Reply:
Yes that's a limitation indeed of the current SSI documentation.
I suggest we open a separate PR adding somewhere in SSI's doc that SSI allows to enable more than APM?